### ISSUE TITLE
Tab order on home screen of project should be configurable

### ISSUE THREAD

**Issue by rmehta** [frappe/gameplan#61]
The home screen of a project should be configurable

- Sometimes people use tasks
- Sometimes they use discussions
- Sometimes you need a readme

 Drag to re-organise tabs on Project?

**Comment by rmehta** [comment]
Bump - always have to spend that extra "click" to move to "discussions" tab if there is not README

**Comment by netchampfaris** [comment]
Navigation is mostly solved via breadcrumbs: https://github.com/frappe/gameplan/commit/a1895af46277856e95cc1f3e12503ab74abf952c
